domains/wackydev.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Turtles127",
+           "email": "turtles127855@gmail.com",
+           "discord": "838934801972723713"
+        },
+    
+        "record": {
+            "CNAME": "wackydev cname wackydev.netlify.app."
```

```suggestion
            "CNAME": "wackydev.netlify.app"
```
Google also flagged this record